README.md

```diff
@@ -210,4 +212,4 @@ GitHub provides a Web UI to make communication process easier
 
 - Mentioning Pull Requests or issues using `#(issue_id)`, Read [this](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue)
 
-- Also, take a look at [this](https://guides.github.com/features/mastering-markdown/), **Markdown or styling** your messages to increase readibilty for PR description and conversations.
\ No newline at end of file
+- Also, take a look at [this](https://guides.github.com/features/mastering-markdown/), **Markdown or styling** your messages to increase readibilty for PR description and conversations.
```

Comment:
@Scramjet911 You have pushed this text a few lines below. I feel like this change is unnecessary. Make sure that your commits are very precise.
Please revert this change and squash.

Reply:
Ok, that was unintentional, I'll fix it

Reply:
#47 is the new PR. 
That issue came up because I edited the readme.md using github edit. It was fixed when I cloned and added the Link locally.

Reply:
I also created a branch for the issue this time.